snowflake/build_and_publish_rpm.sh

```diff
@@ -6,12 +6,17 @@
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $THIS_DIR/version.sh
 TARGET_YUM_REPO=$1
+TARGET_HOST_URL=$2
 
 if [[ -z $TARGET_YUM_REPO ]]; then
   echo "ERROR: {TARGET_YUM_REPO} not specified"
   exit 1
 fi
 
+if [[ -z $TARGET_HOST_URL ]]; then
+  echo "ERROR: {TARGET_HOST_URL} not specified"
+  exit 1
```

Comment:
Now that the usage is getting more complicated, this seems like a good time to add a usage() function.